src/drivers/driver.ts

```diff
@@ -398,6 +398,9 @@ export abstract class CMakeDriver implements vscode.Disposable {
         // Don't fail if CMakeLists.txt was accidentally appended to the sourceDirectory.
         this._sourceDirectory = path.dirname(this._sourceDirectory);
       }
+      if (!(await util.checkFileExists(this._sourceDirectory))) {
```

Comment:
Should this warning go above the "cmakelists.txt" correction above?

Reply:
no, but this should be checking the folder, not a file.  I'm not sure why I didn't see that earlier.